notebooks/examples/configure_check_conditions.ipynb

```diff
@@ -175,13 +175,49 @@
     {
```

Comment:
<p>This is quite a long suite for the purpose of this demonstration, maybe there is a simpler/shorter one?</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/deepchecks/deepchecks/pull/303/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='12'/>